res/enkf/plot_data/ensemble_plot_gen_data.py

```diff
@@ -42,6 +43,10 @@ class EnsemblePlotGenData(BaseCClass):
     )
     _free = ResPrototype("void  enkf_plot_gendata_free(ensemble_plot_gen_data)")
 
+    _copy_realizations = ResPrototype(
+        "void  enkf_plot_gendata_copy_realizations(ensemble_plot_gen_data,double*,int_vector)"
```

Comment:
The people into code formatting will thank you if you inject a few whitespaces between arguments here as well :)

Reply:
Damn auto code formatters dont know how to look inside strings :-) 
Will fix